subsys/bluetooth/controller/Kconfig

```diff
@@ -488,10 +489,38 @@ config BT_CTLR_DTM
 
 config BT_CTLR_DTM_HCI
 	bool "Direct Test Mode over HCI"
+	depends on BT_CTLR_DTM_HCI_SUPPORT
 	select BT_CTLR_DTM
 	help
 	  Enable support for Direct Test Mode over the HCI transport.
 
+comment "BLE Controller debug configuration"
```

Comment:
Why this is moved here to make all the options below grouped under debug?

Reply:
Previously BT_CTLR_ASSERT_HANDLER was already right below "debug". The following options have been moved to be zephyr LL specific:
- BT_CTLR_PROFILE_ISR
- BT_CTLR_DEBUG_PINS

Reply:
@cvinayak, do you suggest to remove the comment completely, or do you want to move the comment to BT_CTLR_PROFILE_ISR and BT_CTLR_DEBUG_PINS?

Reply:
If I understand correctly, this commit is not needed as hardware configurations are already guarded under legacy and split Kconfig options by your previous commits